attestation.md

```diff
@@ -57,7 +61,10 @@ Append the following code to `3-attestation.js`.
 
 ```javascript 
 // build the Attestation object
-const attestation = new Kilt.Attestation(requestForAttestation, attester)
+const attestation = Kilt.Attestation.fromRequestAndPublicIdentity(
+  requestForAttestation,
+  attester
```

Comment:
Request:
Change `attester` to `attester.getPublicIdentity()`, to better illustrate that only a `PublicIdentity` is needed here.

Reply:
Ok, this is good. I have updated.